### PR TITLE
ISPN-12774 Credential store support example for datasource is incomplete

### PR DESCRIPTION
**Pull request by gustavolira**
https://issues.redhat.com/browse/ISPN-12774

### CODE REVIEW

**Comment by oraNod** [comment]
@gustavolira Thanks for the PR! Can we also take this as an opportunity to improve that issue you noted with `<clear-text-credential clear-text="secret"/>` ?

If I understand correctly s/secret/secret1234! https://github.com/infinispan/infinispan/pull/8841/files#diff-04d5620d7c62290027d79c47b8a960d6e541b720335474afdc3c164add6df333R7

**Comment by gustavolira** [comment]
> @gustavolira Thanks for the PR! Can we also take this as an opportunity to improve that issue you noted with `<clear-text-credential clear-text="secret"/>` ?
> 
> If I understand correctly s/secret/secret1234! https://github.com/infinispan/infinispan/pull/8841/files#diff-04d5620d7c62290027d79c47b8a960d6e541b720335474afdc3c164add6df333R7

@oraNod  Sure, I'm on it.

**Comment by gustavolira** [comment]
> > @gustavolira Thanks for the PR! Can we also take this as an opportunity to improve that issue you noted with `<clear-text-credential clear-text="secret"/>` ?
> > If I understand correctly s/secret/secret1234! https://github.com/infinispan/infinispan/pull/8841/files#diff-04d5620d7c62290027d79c47b8a960d6e541b720335474afdc3c164add6df333R7
> 
> @oraNod Sure, I'm on it.

@oraNod done

File: cli/src/main/resources/help/credentials.adoc [hunk 1]
```diff
@@ -67,9 +67,10 @@ EXAMPLES
 --------
 `credentials add dbpassword -c changeme -p "secret1234!"` +
 Creates a new default credential keystore, if does not already exist, and adds an alias of "dbpassword" for a password of "changeme".
-This command also sets "secret1234!" as the password for the credential keystore.
+This command also sets "secret1234!" as the password for the credential keystore, this value should match with the value present in the server configuration like:
+ `<clear-text-credential clear-text="secret1234!"/>`
 
-`credentials ls -p "secret1234!"` +
+`credentials ls -p "secret1234!"` +`
```
Comment (oraNod) [inline cli/src/main/resources/help/credentials.adoc:73]:
```suggestion
`credentials ls -p "secret1234!"` +
```

File: cli/src/main/resources/help/credentials.adoc [hunk 2]
```diff
@@ -67,9 +67,10 @@ EXAMPLES
 --------
 `credentials add dbpassword -c changeme -p "secret1234!"` +
 Creates a new default credential keystore, if does not already exist, and adds an alias of "dbpassword" for a password of "changeme".
-This command also sets "secret1234!" as the password for the credential keystore.
+This command also sets "secret1234!" as the password for the credential keystore, this value should match with the value present in the server configuration like:
```
Comment (oraNod) [inline cli/src/main/resources/help/credentials.adoc:70]:
```suggestion
This command also sets "secret1234!" as the password for the credential keystore, which must match the password in the server configuration:
```

**Review by oraNod — CHANGES_REQUESTED** [review]
@gustavolira Thanks for adding that detail about the password. For the sake of completeness, could you also update the password in the server config example so that it matches the CLI docs? 

https://github.com/infinispan/infinispan/blob/537d58b53f058de1ee9e6e975fc99b0e8d4f4c50/documentation/src/main/asciidoc/topics/config_examples/server_credentials.xml#L7

**Review by oraNod — APPROVED** [review]
LGTM

**Comment by oraNod** [comment]
Merged. Thanks @gustavolira Can you please create a backport for 12.0.x?

**Pull request merged**